apps/user-profile/src/components/modules/UserOnboardingModal/RoleCard/RoleCard.js

```diff
@@ -11,16 +11,22 @@ export default function RoleCard({
     iconImg,
     title,
     subtitle,
-    titletheme,
+    theme,
 }) {
+    const [isActive, setIsActive] = useState(false);
```

Comment:
change  "isActive" to "isSelected". Your using this variable to dynamically apply a class called "selected", may as well keep the name relevant.